src/tools/crossgen2/crossgen2.sln

```diff
@@ -11,46 +11,68 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "ILCompiler.ReadyToRun", "IL
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "ILCompiler.TypeSystem.ReadyToRun", "ILCompiler.TypeSystem.ReadyToRun\ILCompiler.TypeSystem.ReadyToRun.csproj", "{751583CD-E880-49E1-B3E2-8B1990114CAC}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "tibcmgr", "tibcmgr\tibcmgr.csproj", "{89105FB5-C270-4D59-B8A9-7D628113F41D}"
+EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU
```

Comment:
Can you look into why VS created this now?

Only x64/x86 works for crossgen2. "Any CPU" hits the "System.Object not defined" garbage. Everyone starting crossgen2 development is going to hit that and it costs time because "Any CPU" is the VS default. Solution opening in a configuration that doesn't work is a bad experience.

Reply:
At this point I have no way to look into why it created this, as I haven't actually used the solution since I initially created the project. (I've been working in VS Code primarily other than initial project creation as I don't find the VS development environment to be congenial) However, I can easily eliminate the AnyCPU details from the solution.